Source/NameServer/AddressBook.hpp

```diff
@@ -0,0 +1,166 @@
+/*==============================================================================
+ *  AddressBook: A back-end implementation for the POETS sbase & Nameserver
+ *
+ *  Tasks can be *maped* and *linked*:
```

Comment:
tyop

Reply:
Your comment was a typo or something else? (I'm being blind to it it seems...)

Reply:
Sorry, I wasn't trying to poke fun. I realise out-of-context that's not kind. "maped" should be "mapped".